File: java/baiduads-sdk/src/main/java/com/baidu/dev2/api/sdk/manual/oauth/model/RefreshTokenResponse.java
package com.baidu.dev2.api.sdk.manual.oauth.model;


public class RefreshTokenResponse {
    private Integer code;
    private String message;
    private AccessTokenInfo data;

    public Integer getCode() {
        return code;
    }

    public void setCode(Integer code) {
        this.code = code;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public AccessTokenInfo getData() {
        return data;
    }

    public void setData(AccessTokenInfo data) {
        this.data = data;
    }
}

File: python/baiduads-sdk/baiduads_manual/oauth/model/refresh_token_response.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
"""
Copyright (c) 2022 Baidu.com, Inc. All Rights Reserved
"""

from baiduads_manual.oauth.model.access_token_info import AccessTokenInfo


class RefreshTokenResponse:
    """refresh token response data model
    """

    def __init__(self, code, data, message):
        self.code = code
        self.data = data
        self.message = message

    @classmethod
    def dict_to_obj(cls, dict_data):
        """用于将dict转换为对象
        """
        data = None
        if "data" in dict_data:
            data = AccessTokenInfo.dict_to_obj(dict_data['data'])

        return RefreshTokenResponse(dict_data.get("code", None),
                                    data,
                                    dict_data.get("message", None))
